extension/awsproxy/factory_test.go

```diff
@@ -22,8 +22,11 @@ import (
 	"os"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
```

Comment:
Delete this empty line to fix the lint error (it is seeing 4 groups of imports instead of 3 - I agree the error message is very cryptic and wish it could be improved)

Reply:
Great, thanks! Fixed.

Reply:
The suggestion was to delete the extra newline, not to move :P All external dependencies (github.com or go.foo) go in the second group, not the third group so need to move them back (just make sure there is no newline)